content/Solutions/Optimizations/Security.md

```diff
@@ -25,6 +25,7 @@ Follow these best practices to administrate TrueNAS securely.
 * Grant TrueNAS user accounts (local or Directory Services added accounts) access to SSH or console shells only if  that account is explicitly trusted.
```

Comment:
line 23 two-factor authentication, root and administrator don't need to be capitalized